EIPS/eip-2565.md

```diff
@@ -27,19 +27,25 @@ def calculate_multiplication_complexity(base_length, modulus_length):
     words = math.ceil(max_length / 8)
     return words**2
 
-def calculate_iteration_count(exponent_length, exponent):
+def calculate_iteration_count(exponent_length, exponent_head):
```

Comment:
Could we add the parameter comment on `exponent_length` and `exponent_head`? I am a big confused when seeing 
`bit_length = exponent_head.bit_length()` and don't know how it differentiate with `exponent_length`

Reply:
`exponent_length` is in octets, and `bit_length` is in bits. Would `exponent_bit_length` be better?